uint32 mot_1
uint32 mot_2
uint32 mot_3